appbars/src/main/java/com/decathlon/vitamin/compose/appbars/bottomnavigations/VitaminBottomNavigations.kt

```diff
@@ -62,4 +67,48 @@ object VitaminBottomNavigations {
             }
         }
     }
+
+    /**
+     * The bottom navigation allow the user to switch between destinations in an app.
+     * @param actions The [SelectedActionItem] actions of your bottom navigation.
+     * [SelectedActionItem] define the look and the event associated to an item in the bottom navigation
+     * @param modifier The [Modifier] to be applied to this BottomNavigation
+     * @param colors The colors of the background and the content elements in selected and unselected mode
+     */
+    @Composable
+    fun PrimaryImmutable(
+        actions: ImmutableList<SelectedActionItem>,
+        modifier: Modifier = Modifier,
+        colors: BottomNavigationColors = VitaminBottomNavigationColors.primary()
+    ) {
+        BottomNavigation(
```

Comment:
I'm not a big for of code duplication when deprecating methods, IMO the deprecated methods should internally call the new methods as much as possible.

PS: It's an opinion, not of use cases should follow this rule

Reply:
On the principal, I agree but deprecated components should guarantee the exact same behavior than before for all parameters, it isn't always possible. I can simplify this one because I just need to convert the `List` to an `ImmutableList` but it isn't possible for the TopAppbar for example.

Reply:
Yes! Sometimes you can forward the call and sometime you shouldn't because of unexpected behaviors exposed to developers, my point is only for the cases where you can forward the call, I suggest you to do that